samples/nrf9160/cloud_client/Kconfig

```diff
@@ -30,6 +30,10 @@ config CLOUD_PUBLICATION_SEQUENTIAL
 
 endchoice
 
+config POWER_SAVING_MODE_ENABLE
+	bool "Request PSM from cellular network"
+	default n
```

Comment:
`default n` is the actual default, no need to specify it explicitly.

Reply:
Yes, of course. Will change.